src/components/Transactions/Transactions.js

```diff
@@ -79,28 +86,41 @@ const Transactions = ({
             isExpanded={isExpanded}
             length={transactions.length}
           >
-            {transactions.map(subItem => (
-              <InsideWrapperStyled
-                key={subItem.transactionId}
-                length={transactions.length}
-              >
-                <LeftBoxStyled>
-                  <TitleStyled>
-                    {t(`offer-title-${subItem.offerId}`, subItem.offerTitle)}
-                  </TitleStyled>
-                  <SubTitleStyled>
-                    {t(`Paid with`)}{' '}
-                    {subItem.paymentMethod === 'card'
-                      ? t('card')
-                      : subItem.paymentMethod}
-                  </SubTitleStyled>
-                </LeftBoxStyled>
-                <RightBoxStyled>
-                  <IdStyled>{subItem.transactionId}</IdStyled>
-                  <DateStyled>{dateFormat(subItem.transactionDate)}</DateStyled>
-                </RightBoxStyled>
-              </InsideWrapperStyled>
-            ))}
+            {transactions.map(subItem => {
```

Comment:
we can use destructuring on subItem like. map({ paymentMethod, transactionId, ... }) =>